REQUIRE

```diff
@@ -0,0 +1,2 @@
+JuMP
+DistributedArrays
```

Comment:
This should really go into `Krylov.jl`. Perhaps I should stop supporting Julia 0.3.